contact_us/index.html

```diff
@@ -3,40 +3,33 @@
 title: Contact us | St. Anthony's College Kandy
 ---
 <!-- Header -->
-<div class="contact-us container-fluid mt-4">
-  <div class="row google-map">
+<div class="contact-us container-fluid mt-5 pt-5">
+  <div class="row mt-5">
     <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d5005.1003110759!2d80.62775051562666!3d7.322164927958504!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x79aa0d16eecbae25!2sSt.%20Anthony&#39;s%20College!5e0!3m2!1sen!2slk!4v1641754171541!5m2!1sen!2slk" width="100%" height="300vh" style="border:0;" allowfullscreen="" loading="lazy"></iframe>   
   </div>
 </div>
-<div class="main">
-  <section class="mt-5 container-fluid pt-5">
-    <div class="brand px-5 pt-5">
-      <div class="container">
-        <h1 class="title text-roboto">Contact us</h1>
-      </div>
-    </div>
-  </section>
+<div class="main py-5 pl-4">
   <!-- //Header -->
-
   <section class="container pb-5">
+    <h1 class="title">Contact us</h1>
     <div class="row">
-      <div class="col-sm-4 pb-3">
-        <h3 class="font-weight-bold pb-3">Primary Department</h3>
-        <p>Tel: <a href="tel:+94 081 221 3570">081 221 3570</a></p>
-        <p>E-mail: primary@sack.edu.lk</p>
+      <div class="col-sm-4 pb-3 pt-2">
```

Comment:
```suggestion
      <div class="col-sm-4 pb-3">
```
I think we don't need this padding here. Any special reason for adding that?

Reply:
Ok brother I will remove it. No special reason for adding this.